src/main/java/cloud/orbit/web/EmbeddedJettyServer.java

```diff
@@ -229,6 +235,14 @@ public Task start()
             {
                 sslContextFactory.setExcludeCipherSuites(sslExcludedCipherSuites.toArray(new String[0]));
             }
+            if (sslIncludedProtocols != null)
```

Comment:
```java
if (sslIncludedProtocols != null && !sslIncludedProtocols.isEmpty()) 
```
otherwise you are going to set empty collections for setIncludeProtocols and setExcludeProtocols which might not be the same or behaviour may change in the future.